### PR TITLE
Fixes Crushing People Out of Viewing Range

### LINKED ISSUE #16993: [Bug]: Psionic Crush has seemingly unlimited range
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

Psionic was inside the FR's room when he used crush on me, while I was standing in medical reception. I ran all the way to the front of the dining hall and continued being crushed. Another incident involved him using crush while inside operations and I continuing to be crushed all the way at the medbay's reception.

### How to reproduce

1. Get crushed
2. Try to run away

### Round ID

co7-bDD8

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by CampinKiller24**
Fixes #16993 

Range is now checked to make sure you're within range to actually see the person, so you can't just crush people from the other side of the ship anymore. You can still crush people through the wall/being above or below them, as long as you're within normal viewing range.

### CODE REVIEW

**Comment by CampinKiller24** [comment]
!review

**Comment by CampinKiller24** [comment]
!bugfix

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/psionics/abilities/grip.dm [hunk 1]
```diff
@@ -29,19 +29,21 @@
...
 	if(!.)
 		return
-	user.visible_message(SPAN_WARNING("[user] squeezes [user.get_pronoun("his")] hand!"), SPAN_WARNING("You squeeze your hand to tighten the psionic force around [victim]."))
-	log_and_message_admins("[key_name(owner)] has psionically crushed [victim]", owner, get_turf(owner))
-	to_chat(victim, SPAN_DANGER(FONT_HUGE("You are crushed by an invisible force!")))
-	victim.apply_damage(20, DAMAGE_BRUTE, armor_pen = 15, def_zone = BP_HEAD)
-	victim.SetStunned(2)
-	apply_cooldown(user)
-	next_squeeze_time = world.time + 2 SECONDS
+	if (rangecheck < 9)
```
Comment (SleepyGemmy) [inline code/modules/psionics/abilities/grip.dm:39]:
```suggestion
	if(rangecheck < 9)
```

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/psionics/abilities/grip.dm (hunk 1 above)
Comment (Arrow768) [inline code/modules/psionics/abilities/grip.dm:39]: That check can/should be removed as the range is already checked above.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/modules/psionics/abilities/grip.dm (hunk 1 above)
Comment (CampinKiller24) [reply code/modules/psionics/abilities/grip.dm]: Done

**Comment by CampinKiller24** [comment]
!review

**Pull request merged**